tests/unit/test_utils.py

```diff
@@ -524,3 +525,10 @@ def test_manylinux1_check_glibc_version(self):
                 else:
                     # Didn't find the warning we were expecting
                     assert False
+
+
+class Test_user_cache_dir(object):
+
+    def test_return_path_as_str(self):
+        """Test that path is bytes on Python 2 and Unicode on Python 3."""
+        assert isinstance(user_cache_dir('test'), str)
```

Comment:
I was thinking something similar but didn't want to add so complex test when I cannot run tests locally.